### PR TITLE
Gradle 4.5.1

### PR DESCRIPTION
**Pull request by jaredsburrows**
@JakeWharton

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5-bin.zip
```
Comment (MGaetan89) [inline gradle/wrapper/gradle-wrapper.properties]: 4.5.1 is the latest version (https://github.com/gradle/gradle/releases/latest)

Comment (jaredsburrows) [reply gradle/wrapper/gradle-wrapper.properties]: Thanks!

**Comment by jaredsburrows** [comment]
Failed build because of Dokka: https://github.com/Kotlin/dokka/issues/265

**Comment by jaredsburrows** [comment]
Thanks @JakeWharton

**Pull request closed without merging**